backend/battle/templates/battle/base.html

```diff
@@ -196,14 +199,20 @@
             <p class="p_user">{{user}}</p>
         </a>
     </div>
+    {% else %}
+    <div class="user">
+        <a class ="anchor_user" href="{% url 'signup' %}">
+            <img class="img_user" style="width:50px; height:50px" src="https://www.greatmining.com/images/signup.png">
```

Comment:
Please avoid inline styles.
For images, it's fine to add width and height directly as attributes.

Reply:
thanks!